curp/src/server/curp_node.rs

```diff
@@ -222,17 +222,15 @@ impl<C: 'static + Command> CurpNode<C> {
             ) {
                 return Ok(InstallSnapshotResponse::new(self.curp.term()));
             }
-            snapshot
-                .write_all(req.data.as_slice())
-                .await
-                .map_err(|err| {
-                    error!("can't write snapshot data, {err:?}");
-                    err
-                })?;
+            let req_data_len = req.data.len().numeric_cast::<u64>();
```

Comment:
will it be used many times? Or does it make the code more understandable? If not, adding a local variable is a bad idea.

Reply:
the `req.data` is moved when calling `write_all`, so the length must be record before the call.